snooty/parser.py

```diff
@@ -797,13 +797,6 @@ def create_page() -> Tuple[Page, EmbeddedRstParser]:
         for page in pages:
             self._page_updated(page, diagnostic_list)
 
-        # Specify which transformations should be included in semantic postprocessing
-        fn_names: List[str] = ["toctree", "slug-title", "breadcrumbs"]
-        semantic_parse: Dict[str, SerializableType] = self.semantic_parser.run(
-            self.pages, fn_names
-        )
-        self.backend.on_update_metadata(self.prefix, semantic_parse)
```

Comment:
Deleted due to performance issues on Snooty VS Code. Changing which file the editor is focused on ends up causing the extension to lag and be unusable.